packages/components/src/theme/ComposerTheme/Radius/radius.ts

```diff
@@ -17,4 +17,5 @@ export const defaultRadius = {
   'radii-progress-bar': '1.5rem',
   'radii-checkbox': '0px',
   'radii-dropdown': '0px',
+  'radii-badge': '6px',
```

Comment:
Wouldn't this break existing behaviour? I don't think `radii-half` uses `6px`

Reply:
true!